pre_commit_flux/check_flux_helm_values.py

```diff
@@ -66,8 +81,10 @@ def _validateFile(fileToValidate, repos):
                     stderr=subprocess.STDOUT,
                 )
                 if res.returncode != 0:
-                    print(res.stdout)
-                    exit(1)
+                    _collectErrors(
+                        {"source": "helm pull", "message": "\n{0}".format(res.stdout)}
```

Comment:
you should be able to use a format string, something like ```f"\n{res.stdout}"```

Reply:
there are to many pythonic ways i guess :trollface: will change.

Reply:
refactored: https://github.com/tarioch/flux-check-hook/pull/5/commits/de5b285083cabd27f059eb1a9deade0f64f53854#diff-51d4385a76742bdc6b7dba6fbd4d3e84b2aaeb1f9b6b65b77e07a77d8a4dfc40R79